rsopt/configuration/setup.py

```diff
@@ -263,6 +266,9 @@ def _check_setup(cls, setup):
                 raise KeyError(f'{key} in setup block for code-type {code} is not recognized.')
         Setup._check_setup(setup)
 
+    def get_sym_link_targets(self):
+        return {self.setup['input_file']}
```

Comment:
Returning the setup in a new dict, or just returning the dict in self.setup['input_file']? Are you wanting to return a reference to the original, a copy of the original, or wrap in an immutable dict?

Same with line 439 below

Returning different types from the same function, basically

Reply:
This is notation for a set literal. The value at self.setup['input_file'] is returned as the single item in a set.

Reply:
Oh wow. Yes. Yes it is.